shell/platform/android/flutter_main.cc

```diff
@@ -26,7 +27,7 @@
 namespace flutter {
 
 extern "C" {
-#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
+#if FLUTTER_JIT_RUNTIME
 // Used for debugging dart:* sources.
```

Comment:
Is this comment accurate? If it is, then debugging dart:* sources in JIT_RELEASE doesn't seem right.

Reply:
Ahh you're right. I think I confused this with the application kernel file, but it is just the SDK sources and can be left out